Makd.mak

```diff
@@ -487,7 +487,8 @@ clean:
 
 # Phony target to build all packages
 .PHONY: pkg
-pkg: $(patsubst $(PKG)/%.pkg,$O/pkg-%.stamp,$(wildcard $(PKG)/*.pkg))
+PKGS ?= $(wildcard $(PKG)/*.pkg)
```

Comment:
All variables used to accumulate dependencies of a target are in lowercase so far, and maked after the target (`all`, `test`, etc.), I would use simply `pkg`.

I wonder if it should be `+=` instead, as it seems like the most common case will be to want to `$(filter-out )` something instead of building the list from the scratch.